insights/terraform/projects/bcda/main.tf

```diff
@@ -0,0 +1,29 @@
+locals {
+  tags       = { business = "OEDA", application = "bfd-insights", project = "bcda" }
+  project    = "bcda"
+  database   = "bcda"
+  partitions = [{ name = "dt", type = "string", comment = "Approximate delivery time" }]
+}
+
+data "aws_caller_identity" "current" {}
+
+
+## Bucket for the project's data
```

Comment:
Will there be a single bucket for data coming from `prod` and a separate bucket for data coming from `opensbx`?  Or will all data flow into one bucket?

Reply:
We can have a separate Kinesis Firehose Datastream for each environment but the data will all go in the same bucket. The buckets are only distinguished by sensitivity:
* **high**: Contains information which are highly sensitive. PII data has high sensitivity.
* **moderate**: Contains data which are moderately sensitive. DASG logs have moderate sensitivity. Logs are scrubbed not to contain high sensitivity data.